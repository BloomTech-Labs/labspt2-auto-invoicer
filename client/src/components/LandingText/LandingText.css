.landing-text-container {
  background: #8bc34a;
  display: flex;
  justify-content: center;
  min-height: 15vh;
  width: 100%;
}

.landing-text {
  color: #000000;
  display: flex;
  flex-wrap: wrap;
  font-size: 1.6rem;
  justify-content: center;
  max-width: 960px;
  padding: 8px;
  width: 100%;
}
