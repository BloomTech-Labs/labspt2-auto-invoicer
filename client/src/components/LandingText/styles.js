export default theme => ({
  container: {
    paddingTop: 100,
    paddingBottom: 50,
    width: '100%',
    background: '#2d2f31'
  },
  root: {
    width: '100%',
    maxWidth: 700,
    display: 'flex',
    flexDirection: 'column',
    margin: '0 auto'
  },
  headlineText: {
    fontFamily: 'roboto',
    fontSize: 95,
    fontWeight: 500,
    color: 'white',

    [theme.breakpoints.down('md')]: {
      fontSize: 75
    }
  },
  subtext: {
    color: '#8bc34a'
  },
    mobileText: {
    paddingLeft: "5px",
    [theme.breakpoints.up("sm")]: {
      paddingLeft: 0
    }
  }
});