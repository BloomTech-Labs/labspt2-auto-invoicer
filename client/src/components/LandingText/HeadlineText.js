import React from 'react';
import Typography from '@material-ui/core/Typography';
import { withStyles } from '@material-ui/core/styles';

import style from './styles'

const HeadlineText = props => {
  const { classes } = props;
  return (
    <div className={classes.container}>
      <div className={classes.root}>
        <Typography
          variant="h1"
          className={classes.headlineText}
        >
          Invoicing Solved
        </Typography>
        <br />
        <Typography className={classes.subtext} variant="h4">
          Turn sales into cashflow.
        </Typography>
      </div>
    </div>
  );
};

export default withStyles(style)(HeadlineText);
