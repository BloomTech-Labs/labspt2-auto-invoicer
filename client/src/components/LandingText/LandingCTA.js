import React from "react";
import Typography from "@material-ui/core/Typography";
import { withStyles } from "@material-ui/core/styles";

const styles = theme => ({
  container: {
    paddingTop: 20,
    paddingBottom: 20,
    width: "100%",
    background: "#f2f2f2"
  },
  root: {
    width: "100%",
    maxWidth: 700,
    display: "flex",
    flexDirection: "column",
    margin: "0 auto"
  },
  mobileText: {
    paddingLeft: "5px",
    [theme.breakpoints.up("sm")]: {
      paddingLeft: 0
    }
  }
});

const LandingCTA = props => {
  const { classes } = props;
  return (
    <div className={classes.container}>
      <div className={classes.root}>
        <Typography className={classes.mobileText} variant="h4">
          Create a free invoice on the go or sign-up and get unlimited invoices
          today!
        </Typography>
      </div>
    </div>
  );
};

export default withStyles(styles)(LandingCTA);
