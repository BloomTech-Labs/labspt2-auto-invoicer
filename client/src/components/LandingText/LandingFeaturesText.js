import React from "react";

import { ReactComponent as InvoicesSVG } from "../../assets/undraw_printing_invoices_5r4r.svg";
import { ReactComponent as ChartsSVG } from "../../assets/undraw_finance_0bdk.svg";
import Typography from "@material-ui/core/Typography";
import { withStyles } from "@material-ui/core/styles";

const styles = {
  container: {
    display: "flex",
    justifyContent: "center",
    width: "100%",
    background: "#ffffff"
  },
  features: {
    display: "flex",
    flexWrap: "wrap",
    fontSize: "1.6rem",
    maxWidth: 1080,
    padding: "100px 0",
    width: "100%"
  },
  top: {
    alignItems: "center",
    display: "flex",
    flexWrap: "wrap",
    justifyContent: "space-between",
    width: "100%"
  },
  topText: {
    maxWidth: 360,
    padding: "8px",
    width: "100%"
  },
  bottom: {
    alignItems: "center",
    display: "flex",
    flexWrap: "wrap",
    justifyContent: "space-between",
    marginTop: 40,
    width: "100%"
  },
  bottomText: {
    maxWidth: 460,
    padding: "8px",
    width: "100%"
  }
};

const LandingFeaturesText = props => {
  const { classes } = props;
  return (
    <div className={classes.container}>
      <div className={classes.features}>
        <div className={classes.top}>
          <InvoicesSVG />
          {/* <img src="/assets/placeholder-460x300.png" /> */}
          <div className={classes.topText}>
            <Typography variant="h4">
              Create, send and track invoices in minutes.
            </Typography>
            <br />
            <Typography variant="body1">
              Create customer profiles, save recurring items and invoice your
              customers from anywhere, so you can get paid faster!
            </Typography>
          </div>
        </div>
        <div className={classes.bottom}>
          <div className={classes.bottomText}>
            <Typography variant="h4">
              Comprehensive dashboard provides all your data at a glance.
            </Typography>
            <br />
            <Typography variant="body1">
              Gain insight into your best customers, including those who need a
              friendly nudge. Forecast your business cashflow so you can plan
              accordingly.
            </Typography>
          </div>
          <ChartsSVG />
          {/* <img src="/assets/placeholder-460x250.png" /> */}
        </div>
      </div>
    </div>
  );
};

export default withStyles(styles)(LandingFeaturesText);
