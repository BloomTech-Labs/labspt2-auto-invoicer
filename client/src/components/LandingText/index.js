import React from "react";

import "./LandingText.css";

const LandingText = props => {
  return (
    <div className="landing-text-container">
      <div className="landing-text">
        <p>
          Landing Text: Lorem ipsum dolor sit amet, consectetur adipiscing elit,
          sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
          Pellentesque dignissim enim sit amet venenatis urna cursus eget nunc.
        </p>
      </div>
    </div>
  );
};

export default LandingText;
