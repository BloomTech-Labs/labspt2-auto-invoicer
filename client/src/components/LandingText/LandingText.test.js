import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import LandingText from "../LandingText";

describe("<LandingText />", () => {
  it("renders Landing Text", () => {
    const { asFragment } = render(<LandingText />);
    expect(asFragment()).toMatchSnapshot();
  });
});
