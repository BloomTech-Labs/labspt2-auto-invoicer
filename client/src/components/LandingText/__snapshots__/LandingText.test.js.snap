// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingText /> renders Landing Text 1`] = `
<DocumentFragment>
  <div
    class="landing-text-container"
  >
    <div
      class="landing-text"
    >
      <p>
        Landing Text: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Pellentesque dignissim enim sit amet venenatis urna cursus eget nunc.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
